src/NuGet.Core/NuGet.Commands/NuGet.Commands.csproj

```diff
@@ -7,7 +11,7 @@
     <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
     <TargetFramework />
     <NoWarn>$(NoWarn);CS1591;CS1574;CS1573;CS1584;CS1658</NoWarn>
-    <NoWarn Condition="'$(TargetFramework)' == '$(NetStandardVersion)'">$(NoWarn);CS1998</NoWarn>
+    <NoWarn Condition=" $(TargetFramework.StartsWith('netstandard')) OR $(TargetFramework.StartsWith('netcoreapp')) ">$(NoWarn);CS1998</NoWarn>
```

Comment:
Does IsCore not work here?

Reply:
IsCore doesn't work.
The assignment of property `IsCore `is in common.targets.
And the NuGet.commands.csproj imports the common.targets at the end, that's after the `NoWarn`.

Reply:
We can probably move IsCore to common.props...I can look into that clean-up.